tests/test_fpp.py

```diff
@@ -1,20 +1,242 @@
 import unittest
 import pandas as pd
 import matplotlib.pyplot as plt
-from statwrap.fpp import apply_pd_changes, sd
+from statwrap.fpp import apply_pd_changes, sd, standard_units, average, r, rms_size, var, sd_plus, var_plus
 
+
+class TestCorrelation(unittest.TestCase):
+
+    def setUp(self):
+        self.x1 = [0, 1, 1]
+        self.y1 = [2, -9, 2]
+        self.x2 = [1, 2, 3, 4, 5]
+        self.y2 = [5, 4, 3, 2, 1]
+        self.x3 = [1, 2, 3]
+        self.y3 = [4, 5, 6]
+
+    def test_correlation_basic(self):
+        result = r(self.x1, self.y1)
+        expected = -0.5
+        self.assertAlmostEqual(result, expected)
+
+    def test_correlation_inverse(self):
+        result = r(self.x2, self.y2)
+        expected = -1.0
+        self.assertAlmostEqual(result, expected)
+
+    def test_correlation_perfect(self):
+        result = r(self.x3, self.y3)
+        expected = 1.0
+        self.assertAlmostEqual(result, expected)
+
+    def tearDown(self):
+        pass
+
+class TestAverage(unittest.TestCase):
+
+    def setUp(self):
+        self.numbers_array = [0, 5, -8, 7, -3]
+        self.numbers_multiple = [1, 2, 3, 4, 5]
+        self.single_value = [42]
+        self.mixed_types = [1, 2.5, 3, 4.5]
+
+    def test_average_array(self):
+        result = average(self.numbers_array)
+        expected = 0.2
+        self.assertAlmostEqual(result, expected)
+
+    def test_average_multiple_args(self):
```

Comment:
can you include a comment to make the difference between this and the test on 43 more apparent, where this one unpacks a list and the other doesn't?